.github/workflows/release.yml

```diff
@@ -53,10 +52,7 @@ jobs:
         with:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
           tags: |
-            type=ref,event=branch
             type=ref,event=tag
```

Comment:
This can be removed too. So it become same as relay release: https://github.com/paralus/relay/blob/64d912141e4198090f136cf2735f35ec3a08bc5f/.github/workflows/release.yml#L53-L57
By default it will set image tag based on github tag.